File: showcase/grade-analyze.php
<?php
// Ask the user to input a numerical grade
$grade = readline("Enter your grade: ");

// Convert the grade to integer
$grade = (int)$grade;

// Check the grade and print the corresponding letter grade
if ($grade >= 90) {
    echo "A";
} elseif ($grade >= 80) {
    echo "B";
} elseif ($grade >= 70) {
    echo "C";
} elseif ($grade >= 60) {
    echo "D";
} else {
    echo "F";
}

echo "\n";

File: showcase/grade-analyze.c
#include <stdio.h>

int main()
{
  int grade;

  // Ask the user to input a numerical grade
  printf("Enter the grade: ");
  scanf("%d", &grade);

  // Check the grade and print the corresponding letter grade
  if (grade >= 90)
  {
    printf("A\n");
  }
  else if (grade >= 80 && grade <= 89)
  {
    printf("B\n");
  }
  else if (grade >= 70 && grade <= 79)
  {
    printf("C\n");
  }
  else if (grade >= 60 && grade <= 69)
  {
    printf("D\n");
  }
  else
  {
    printf("F\n");
  }

  return 0;
}